external/gtest.cmake

```diff
@@ -54,12 +54,12 @@ set(BUILD_GMOCK   OFF CACHE BOOL "prevents building gmock"   FORCE)
 add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR})
 
 if(MSVC)
-  set(gtest_LIBRARIES ${googletest_BINARY_DIR}/googletest/$<$<CONFIG:Release>:Release>$<$<CONFIG:Debug>:Debug>/${CMAKE_STATIC_LIBRARY_PREFIX}gtest$<$<CONFIG:Debug>:d>${CMAKE_STATIC_LIBRARY_SUFFIX})
+	set(gtest_LIBRARIES ${EP_BASE}/lib/$<$<CONFIG:Release>:Release>$<$<CONFIG:Debug>:Debug>/${CMAKE_STATIC_LIBRARY_PREFIX}gtest$<$<CONFIG:Debug>:d>${CMAKE_STATIC_LIBRARY_SUFFIX})
 else()
   if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
     set(DEBUG_POSTFIX d)
   endif()
-  set(gtest_LIBRARIES ${googletest_BINARY_DIR}/googletest/${CMAKE_STATIC_LIBRARY_PREFIX}gtest$<$<CONFIG:Debug>:d>${CMAKE_STATIC_LIBRARY_SUFFIX})
+  set(gtest_LIBRARIES ${EP_BASE}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}gtest$<$<CONFIG:Debug>:d>${CMAKE_STATIC_LIBRARY_SUFFIX})
```

Comment:
@dkeeney I have to make this "hack" because the `libgtest.a` is not generated as it used to (chaned in the new version?) but now in `lib/`. Is there a nicer way that avoids this?

Reply:
Hmmm, are you sure.  ${googletest_BINARY_DIR} is the build folder for gtest, created by the download thing.  ${EP_BASE} would be one level up.

So was at ${googletest_SOURCE_DIR}/googletest and now at ${googletest_SOURCE_DIR}/lib ?

Reply:
Also, did they move the includes too?

Reply:
> Hmmm, are you sure. ${googletest_BINARY_DIR} is the build folder for gtest [...]

I'm sure it's now in `$EP_BASE/lib` as the build was failing and I used `find` to get the file's location. Not sure why it's being put there, though. 

> So was at ${googletest_SOURCE_DIR}/googletest and now at ${googletest_SOURCE_DIR}/lib ?

no, the diff is from worked-before to works-now. So it used to be at `${googletest_BINARY_DIR}/googletest/` and now resides at `EP_BASE/lib`

> did they move the includes too?

seems not, the code works ok as-is now.

Reply:
> What I do is run a build and see where the lib ends up. then go change the link path accordingly.

this is what I did, so probably ok.